### DISCUSSION TITLE
[Customizing typography in Zed] Can we change the Line height value?

### DISCUSSION THREAD

**Discussion by florinpatrascu** [zed-industries/zed#6522] [Feature Requests]
Hello and thank you so much for Zed, it looks like this will become my favorite editor =)

Since I am one of those developers cursed to spend hours on customizing everything that requires a good font, ligatures, color, aspect ratio etc. everything related or closer to typography, I am still searching and can't find how I can change the line height of the editor? Is it possible?

Thank you,
ッFlorin

**Comment by fdionisi** [comment]
@florinpatrascu, unfortunately, it's not currently possible to adjust line height. But you can 👍 to [this issue](https://github.com/zed-industries/community/issues/304), so the team is informed and can prioritise the feature. 🙏

**Comment by JosephTLyons** [comment]
We now support disabling ligatures (v0.79.0), as we as some other OpenType features


The following setting should disable ligatures:

```json

{
  "buffer_font_features": {
    "calt": false
  }
}
```

https://docs.zed.dev/configuration/configuring-zed#font-features

This doesn't cover all of the requests in this Discussion, but just wanted to point it out.

**Comment by JosephTLyons** [comment]
Line height is now configurable in Zed.

**Comment by sgmonda** [comment]
Hi! I can edit line-height for buffers and terminal, but is it possible to edit line-height for UI? I'm not able to find a setting.

**Reply by ettyxne** [reply]
how u imagine this?

**Reply by sgmonda** [reply]
> how u imagine this?

This way 🙂: https://github.com/zed-industries/zed/pull/51271
